src/wallet/coinselection.h

```diff
@@ -254,6 +254,10 @@ struct SelectionResult
 private:
     /** Set of inputs selected by the algorithm to use in the transaction */
     std::set<COutput> m_selected_inputs;
+    /** The target the algorithm selected for. Note that this may not be equal to the recipient amount as it can include non-input fees */
+    CAmount m_target;
```

Comment:
I wish we had canonical terms to consistently disambiguate the selection target and the recipient output total, especially because it additionally sometimes includes cost of change and a minimum change.

What do you all think about something along the lines of:
- `recipients_total`: sum of recipient output values (excluding change)
- `selection_target`: the value the coin selection needs to raise to pay for the payload including change for algorithms that allow change

Do we need another term for `selection_target` without change?